prebuild/README.md

```diff
@@ -2,9 +2,16 @@
 
 ## Get Debian base boxes
 
+### For the 64 bit box
+
 ```bash
 vagrant box add debian/contrib-jessie64
 ```
+### For the 32 bit box
+
+```bash
+vagrant box add bento/debian-8.6-i386
```

Comment:
As an update, the Debian cloud team is not interested in maintaining 32 bit images of Debian for Vagrant, so I suggest to stick with the Bento images.